digi/xbee/models/options.py

```diff
@@ -403,3 +403,63 @@ def calculate_discovery_value(protocol, options):
 
 DiscoveryOptions.lookupTable = {x.code: x for x in DiscoveryOptions}
 DiscoveryOptions.__doc__ += utils.doc_enum(DiscoveryOptions)
+
+
+@unique
+class UserDataRelayInterface(Enum):
+    """
+    Enumerates the different interfaces for the :class:`.UserDataRelayRequest`
```

Comment:
I suppose having these links going out to classes that don't exist is a bad idea. Should I remove them and just document that these values are associated with 0x2D/0xAD API frames?